release-notes/2.0/2.0.3.md

```diff
@@ -0,0 +1,104 @@
+# .NET Core 2.0 November Update - November 14, 2017
+
+.NET Core 2.0.3 and SDK 2.0.3 are available for download and usage in your environment.
+
+* [Getting Started](https://www.microsoft.com/net/core/)
+* [Downloads](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.3-download.md)
+
+The blog roundup
+
+* [.NET Core](https://blogs.msdn.microsoft.com/dotnet/2017/08/14/announcing-net-core-2-0/)
+* [NETStandard2](https://blogs.msdn.microsoft.com/dotnet/2017/08/14/announcing-net-standard-2-0/)
+* [ASP.NET Core](https://blogs.msdn.microsoft.com/webdev/2017/08/14/announcing-asp-net-core-2-0/)
+* [EF Core](https://blogs.msdn.microsoft.com/dotnet/2017/08/14/announcing-entity-framework-core-2-0/)
+
+The .NET Core SDK 2.0.0 includes .NET Core 2.0.0 Runtime so downloading the runtime packages separately is not needed when installing the SDK. After installing the .NET Core SDK 2.0.0, the following command will show that you're running version `2.0.3` of the tools.
+
+`dotnet --version`
+
+Your feedback is important and appreciated. We've created [dotnet/core #???](https://github.com/dotnet/core/issues/???) for your questions and comments.
+
+## Azure AppServices
+
+Deployment of .NET Core 2.0 support on Azure AppServices is in process. Because AppServices is a high availability service, the deployment is carfully staged across regions over a period of time. Deployment will begin in teh West US 2 and North Central US regions with remaining regions following over few days.
+
+## .NET Core 2.0 Highlights
+
+### Security
+
+### Docker Images
+
+The [.NET Core Docker images](https://hub.docker.com/r/microsoft/dotnet/) have been updated for this release. Look for the 2.0.3 images.
+
+## ASP.NET Core
+
+Please see the [ASP.NET Core release notes](https://github.com/aspnet/Home/releases/tag/2.0.3) and [ASP.NET Core](https://blogs.msdn.microsoft.com/webdev/2017/08/14/announcing-asp-net-core-2-0/) for highlights on work from ASP.NET Core, MVC, Entity Framework Core and others.
+
+## Fixes in the November 2017 Update
+
+## CLI
+
+* [`[8a60d17]`](https://github.com/dotnet/cli/commit/8a60d17) Update F# compiler to match VS
+* [`[6ef8af7]`](https://github.com/dotnet/cli/commit/6ef8af7) update fsharp compiler for preview release
+* [`[4a2d3fa]`](https://github.com/dotnet/cli/commit/4a2d3fa) Fix up roslyn satellite assembly handling to match new insertion mechanism
+* [`[6ccff28]`](https://github.com/dotnet/cli/commit/6ccff28) Update F# compiler
+* [`[fb67e25]`](https://github.com/dotnet/cli/commit/fb67e25) MSBuild 15.5.110
```

Comment:
@leecow I don't think this and other PRs are part of the 2.0.3 SDK but rather in the sdk shipping with the 15.5 version (2.1.1 IIRC)

Reply:
@livarcocc to confirm